examples/README.md

```diff
@@ -1,9 +1,15 @@
+<<<<<<< HEAD
 [<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="200"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
+=======
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-primary-logo.svg" width="400"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
+>>>>>>> 6002e1c... correct broken README.mds
```

Comment:
conflicts in files not resolved

Reply:
resolved in 53d269e. Updated to permanent logo also.